File: labaTRPOtransport/Implement/Station.java
package com.company;

public class Station {
    private String name;
    private Point coordinates;

    public Station(){}

    public Station(String name, Point coordinates){
        this.setName(name);
        this.setCoordinates(coordinates);
    }


    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public Point getCoordinates() {
        return coordinates;
    }

    public void setCoordinates(Point coordinates) {
        this.coordinates = coordinates;
    }
}

File: lab3/FIrstUMLLab/Station.cs
namespace FIrstUMLLab
{
    class Station
    {
        public string Name { get; private set; }
        public Point Coordinates { get; private set; }

        public Station(string name, Point coordinates)
        {
            this.Name = name;
            this.Coordinates = coordinates;
        }
    }
}
